pyproject.toml

```diff
@@ -28,3 +28,21 @@ requires = [
 
     "scipy>=0.19.1",
 ]
+
+[tool.black]
+line-length = 88
+exclude = '''
+
+(
+  /(
+      \.eggs         # exclude a few common directories in the
+    | \.git          # root of the project
+    | \.mypy_cache
+    | examples
```

Comment:
If we use black for code, it might make sense to use it for examples as well maybe? Because if we don't use consistent formatting for all python it can be confusing for contributors.

Reply:
In yesterday's meeting we discussed that applying black to examples could potentially lead to big /scary code sections. No strong opinion on my side, although it's true that we've been following slightly different conventions for examples. Typically in the notebook-style ones, we write the import at the beginning of each cell instead of at the top of the file.

Reply:
For me it comes down to a UX thing. When reading an example and the code block uses up a significant portion of the vertical space, does this make the example harder to read?